src/cortexpy/tesserae.py

```diff
@@ -283,57 +283,57 @@ def __recurrence(self, query, panel, lsize_l, l1, max_r, pos_max, state_max, who
                     tb_base = who_max * 10 + state_max + pos_max / self.tb_divisor
                     for pos_seq in range(1, len(target) + 1):
                         # Match
-                        self.vt_m[seq][pos_seq][1 - pos_target % 2] = vt_m_base
+                        self.vt_m[1 - pos_target % 2][seq][pos_seq] = vt_m_base
```

Comment:
Actually, I think encapsulating these matrices in a class would simplify things and make these sort of changes easier and less error prone going forward.